src/web/src/sse.cc

```diff
@@ -181,7 +181,7 @@ std::string Sse::loadAssetFromDatacenter()
     }
     log_debug("=== end ===");
   }
-  //Get all devices without location
```

Comment:
WARNING: any modification to SSE here MUST also be applied to etn-ipm2-rest similar files.
Otherwise, this will be overriden by that last in the resulting images!
However, the below resemble to a backport of another PR. @jeffdegott plz check and report, thx.

Reply:
I confirm. etn-ipm2-rest SSE code (etn_sse.ecpp, sse.cc)  *must* be synchronized with fty-rest equivalent.

Reply:
@jeffdegott, @aquette The modification from this file are already on etn-ipm2-rest. I also discover that HOTFIX_PQSWMBT_3166 is on etn-ipm2-rest but not on fty-rest

Reply:
Exact. The hotfix should be backported to the fty repo after release.